dns/pfSense-pkg-bind9/files/usr/local/pkg/bind.inc

```diff
@@ -43,9 +43,14 @@ function bind_zone_validate($post, &$input_errors) {
 		$_POST['mail'] = preg_replace("/@/", ".", $post['mail']);
 	}
 
+	if (isset($_POST['rpz']) && $_POST['rpz'] == "on") {
+		if ($_POST['type'] !== "master" && $_POST['type'] !== "slave") {
```

Comment:
In this line and the line above, please put the individual tests with multiple components inside parenthesis, for example:
`if (isset($_POST['rpz']) && ($_POST['rpz'] == "on")) {`
and
`if (($_POST['type'] !== "master") && ($_POST['type'] !== "slave")) {`